engine/standalone/util/src/main/scala/pl/touk/nussknacker/engine/standalone/utils/typed/TypedMapUtils.scala

```diff
@@ -0,0 +1,30 @@
+package pl.touk.nussknacker.engine.standalone.utils.typed
+
+import io.circe.{Json, JsonObject}
+import pl.touk.nussknacker.engine.api.typed.TypedMap
+
+import scala.collection.JavaConverters._
+
+object TypedMapUtils {
+
+    def jsonToTypedMap(json: Json): TypedMap =
+      jsonObjectToTypedMap(json.asObject.getOrElse(JsonObject.empty))
+
+    private def jsonToAny(json: Json): Any = {
+      json.fold(
+        jsonNull = null,
+        jsonBoolean = identity,
+        jsonNumber = d => BigDecimal(d.toDouble), //prevent scientific notation
```

Comment:
Can u use `java.math.BigDecimal` instead? We have very limited support for scala types (if we have any?). In the other hand, spel have very nice support for standard java types - for example it has in flight conversions between bigdecimal and integer and many others